### DISCUSSION TITLE
是否可以支持副屏？（Is it possible to support the secondary screen?）

### DISCUSSION THREAD

**Discussion by iceriny** [zhongyang219/TrafficMonitor#1057] [Ideas]
更新win11后，我把它放在了主屏幕的最左边，但是最近系统更新后，最左边的位置是一个小部件，所以我把它放在了右边——这会导致任务栏不平衡，不美观。 . 那么我们可以添加一个显示在副屏上的开关吗？

(After updating win11, I put it on the leftmost side of the home screen, but after the recent system update, the leftmost position is a widget, so I put it on the right side - this will cause the taskbar to be unbalanced and unsightly. . So can we add a switch displayed on the secondary screen?)

**Comment by DevMass** [comment]
I would really appreciate it if it can be done.
